twitter.css

```diff
@@ -0,0 +1,95 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr;
```

Comment:
Not exactly the Twitter grid layout, but it's a nice solution and works well in terms of responsivity.